### PR TITLE
Make style for selection configurable

### PR DESCRIPTION
**Pull request by JakobMiksch**
- Add a property of a default selection style
- It can be overwritten by layer specific selection styles

### CODE REVIEW

**Comment by chrismayer** [comment]
Thanks for this.

Isn't it possible to instantiate the `SelectInteraction` always with 

```js
selectClick = new SelectInteraction({
    layers: [layer],
    style: selectStyle
});
```
and in case `selectStyle` is undefined the default style of the `SelectInteraction` will be applied? So the if/else could be removed?

**Comment by JakobMiksch** [comment]
@chrismayer true. I removed the if/else  in the latest commit.

File: app-starter/static/app-conf.json [hunk 1]
```diff
@@ -31,6 +31,13 @@
     "history": true
   },
 
+  "defaulSelectStyle": {
```
Comment (chrismayer) [inline app-starter/static/app-conf.json:34]:
```suggestion
  "defaultSelectStyle": {
```

File: docs/wegue-configuration.md [hunk 2]
```diff
@@ -25,6 +25,7 @@ This describes the Wegue application configuration, which is modelled as JSON do
 | **mapLayers**      | Array of map layer configuration objects | See [mapLayers](map-layer-configuration) |
 | projectionDefs     | Array of CRS / projection definition objects compatible to proj4js | See [projectionDefs](wegue-configuration?id=projectiondefs) |
 | tileGridDefs       | Array of tile grid definition objects | See [tileGridDefs](wegue-configuration?id=tilegriddefs) |
+| defaulSelectStyle  | The default style for a selected feature | see [style](map-layer-configuration?id=style-for-vectorlayers) |
```
Comment (chrismayer) [inline docs/wegue-configuration.md:28]:
```suggestion
| defaultSelectStyle  | The default style for a selected feature | see [style](map-layer-configuration?id=style-for-vectorlayers) |
```

File: src/components/ol/Map.vue [hunk 3]
```diff
@@ -163,9 +164,21 @@ export default {
...
         // if layer is selectable register a select interaction
         if (lConf.selectable) {
+          // check if a style is provided in the appConfig
+          let selectStyle;
+          if (lConf.selectStyle) {
+            // layer specific select style
+            selectStyle = OlStyleFactory.getInstance(lConf.selectStyle);
+          } else if (appConfig.defaulSelectStyle) {
+            // default select style for all layers
+            selectStyle = OlStyleFactory.getInstance(appConfig.defaulSelectStyle)
```
Comment (chrismayer) [inline src/components/ol/Map.vue:174]:
```suggestion
            selectStyle = OlStyleFactory.getInstance(appConfig.defaultSelectStyle)
```

**Review by chrismayer — CHANGES_REQUESTED** [review]

**Comment by JakobMiksch** [comment]
@chrismayer like discussed I removed the `defaultSelectionStyle`

**Review by chrismayer — APPROVED** [review]
LGTM now, thanks!

**Pull request merged**